lib/port_scanner.py

```diff
@@ -46,5 +38,4 @@ def scan_port_range(host, start_port, end_port, n_threads):
     for thread in threads:
         thread.join()
 
-    print(f"[{Color.LIGHT_GREEN}!{Color.RESET}] Port scan finished")
```

Comment:
I thought this was a nice touch to let the user know the scan is finished

Reply:
okay, you're right